test/Parser.spec.ts

```diff
@@ -57,11 +57,40 @@ describe('parser', () => {
   it('should properly emit metadata event when metadata has been updated', async () => await new Promise((resolve) => {
     expect.hasAssertions();
 
-    const radio = new Parser({ autoUpdate: false, notifyOnChangeOnly: true, url: 'https://live.hunter.fm/80s_high' });
+    const radio = new Parser({ autoUpdate: false, notifyOnChangeOnly: [], url: 'https://live.hunter.fm/80s_high' });
```

Comment:
This change is no longer needed then and this test will check that everything works as before